### DISCUSSION TITLE
Getting error with Add-Migration

### DISCUSSION THREAD

**Discussion by MahmoudX10** [jasontaylordev/CleanArchitecture#796] [Q&A] [answered]
I get the following error when I run Add-Migration

`Unable to resolve service for type 'Microsoft.EntityFrameworkCore.Migrations.IMigrator'. This is often because no database provider has been configured for this DbContext. A provider can be configured by overriding the 'DbContext.OnConfiguring' method or by using 'AddDbContext' on the application service provider. If 'AddDbContext' is used, then also ensure that your DbContext type accepts a DbContextOptions<TContext> object in its constructor and passes it to the base constructor for DbContext.`

**Comment by ramax495** [comment, marked as answer]
Check appsettings.json. It should be `"UseInMemoryDatabase": false`

**Reply by MahmoudX10** [reply]
Thank you so much it solved my issue
